scripts/spack/packages/serac/package.py

```diff
@@ -82,10 +82,8 @@ class Serac(CachedCMakePackage, CudaPackage):
     depends_on('python', when="+devtools")
     depends_on('py-sphinx', when="+devtools")
 
-    # Workaround for concretizer bug
-    #depends_on("mfem+sundials", when="+sundials")
-    #depends_on("sundials~shared", when="+sundials")
-    depends_on("sundials~shared+hypre+monitoring")
+    depends_on("sundials~shared+hypre+monitoring~examples-c~examples-f77~examples-install",
```

Comment:
YIKES.

Reply:
reason: sundials examples started failing to link at sometime on toss3 clang.  i do not know why.  we also do not need them so i turned them off via the given sundials variants: ~ for off + for on

Reply:
@white238 Just ran into this issue (twice; once with Serac and again with Smith). Has it been reported upstream to Spack and/or Sundials?

Reply:
it has not.. should but was busy to get this out the door.

Reply:
Also I'm still not clear what was the cause of this.  It used to work and as far as I know nothing changed to cause this. Plus we certainly don't depend on any of their examples and their library works just fine.

Reply:
Agreed; it used to work for us, too.

Reply:
I'm supposed to review the updates to MFEM's Spack package and I'm not sure what to tell them re: Sundials, so I'm doing some diagnostic work to see if I can get more info.